PKHeX.WinForms/Resources/text/lang_es.txt

```diff
@@ -37,7 +37,7 @@ SAV_PokedexORAS=Editor de Pokédex (ORAS)
 SAV_PokedexSM=Editor de Pokédex 
 SAV_PokedexSWSH=Pokédex Editor
```

Comment:
`Editor de Pokédex (SWSH)`

Reply:
I might want to check these, since I don't really think adding the games are necessary there?